app/content-panel/blogs/create/page.tsx

```diff
@@ -1,15 +1,33 @@
+'use client';
+
 /* eslint-disable jsx-a11y/label-has-associated-control */
 
+import React from 'react';
+import { useSession } from 'next-auth/react';
 import styles from './createArticlePage.module.scss';
 import PublishArticleForm from '../../../../components/module-components/blog/PublishArticleForm';
 
+type UserRole = 'content_manager' | 'admin';
+
 const PostArticle = () => {
+  const { data: session, status } = useSession();
+
+  if (status === 'loading') {
+    return <div>Loading....</div>;
+  }
```

Comment:
Usually when we are on a page example here blogs, and we show indication of loading it should indicate that the posts are loading not the status of the logged in user.
I think it would be helpfull if you check what we get from useSession